doc/source/PreConfiguredSites.rst

```diff
@@ -100,22 +100,22 @@ For ``spack-stack-1.7.0`` with Intel, load the following modules after loading m
 
```

Comment:
In the non-compiler specific block above this, there is a line with ```module load python/3.9.2```, but I get this message when try to load python:
```
Lmod has detected the following error:  The following module(s) are unknown: "python/3.9.2"
```
Is this the right python version?

Reply:
no, that's wrong - no longer needed.

Reply:
fixed